### PR TITLE
chore(CHANGELOG): update to v4.9.0

### PR DESCRIPTION
**Pull request by nikku**
What is missing is the actual library version bumps.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,39 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* `FEAT`: add basic what's new communication ([#2303](https://github.com/camunda/camunda-modeler/issues/2303))
+* `FEAT`: remove ambiguous `+` button ([#2312](https://github.com/camunda/camunda-modeler/issues/2312), [#2293](https://github.com/camunda/camunda-modeler/issues/2293))
+* `FEAT`: await loading of plug-in provided, injected styles ([`e038073a`](https://github.com/camunda/camunda-modeler/commit/e038073a76f5e9d150724fb35b15f5dd059da422))
+* `FIX`: correct opening of files in already running editor instance ([#2268](https://github.com/camunda/camunda-modeler/issues/2268))
+* `CHORE`: migrate to GitHub actions ([#2245](https://github.com/camunda/camunda-modeler/issues/2245), [#2242](https://github.com/camunda/camunda-modeler/issues/2242))
+
+### BPMN
+
+* `FEAT`: support [bpmn-in-color](https://github.com/bpmn-miwg/bpmn-in-color)
+* `FEAT`: support setting `variables` and `local` property via element templates independently ([#2334](https://github.com/camunda/camunda-modeler/issues/2334)
```
Comment (barmac) [inline CHANGELOG.md:24]:
```suggestion
* `FEAT`: support setting `variables` and `local` property via element templates independently ([#2334](https://github.com/camunda/camunda-modeler/issues/2334))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,39 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
-* `FIX`: allow to connect call activity with message flow ([#942](https://github.com/camunda/camunda-modeler/issues/942))
+## 4.9.0
+
+### General
+
+* `FEAT`: support Zeebe 1.1 / Camunda Cloud 1.1 ([#2319](https://github.com/camunda/camunda-modeler/issues/2319), [#2298](https://github.com/camunda/camunda-modeler/issues/2298), [#2297](https://github.com/camunda/camunda-modeler/issues/2297), [#2296](https://github.com/camunda/camunda-modeler/issues/2296))
+* `FEAT`: improve status bar ([#2318](https://github.com/camunda/camunda-modeler/issues/2318), [#2303](https://github.com/camunda/camunda-modeler/issues/2303))
+* `FEAT`: add basic what's new communication ([#2303](https://github.com/camunda/camunda-modeler/issues/2303))
+* `FEAT`: remove ambiguous `+` button ([#2312](https://github.com/camunda/camunda-modeler/issues/2312), [#2293](https://github.com/camunda/camunda-modeler/issues/2293))
+* `FEAT`: await loading of plug-in provided, injected styles ([`e038073a`](https://github.com/camunda/camunda-modeler/commit/e038073a76f5e9d150724fb35b15f5dd059da422))
```
Comment (barmac) [inline CHANGELOG.md:17]: How about if we link to the PR?

```suggestion
* `FEAT`: await loading of plug-in provided, injected styles ([`#2281`](https://github.com/camunda/camunda-modeler/pull/2281))
```

**Comment by barmac** [comment]
Two minor comments. This release is 🧨

**Comment by nikku** [comment]
Thanks for your review. Hints appreciated.

**Pull request merged**